README.md

```diff
@@ -66,6 +66,9 @@ versions of the course, run `MDBOOK_BOOK__LANGUAGE=xx mdbook build -d book/xx`
 where `xx` is the ISO 639 language code (e.g. `da` for the Danish translation).
 [TRANSLATIONS.md](TRANSLATIONS.md) contains further instructions.
 
+> **Note**·On·Windows,·you·need·to·enable·symlinks
```

Comment:
Oh wow, I managed to copy the center-dots from the `dprint` error message... This is what I meant:

```suggestion
> **Note** On Windows, you need to enable symlinks
```